doc/src/explore/devnet.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Experiment with Sui DevNet
+---
+
+Welcome to the beginnings of the Sui DevNet. It exists now to gain operational experience with the Sui software in a public setting. The Sui DevNet currently consists of:
+
+* A four-validator network with all nodes operated by Mysten Labs. Clients send transactions and read requests via this endpoint: http://gateway.devnet.sui.io/
+* A public network [Sui Explorer](https://github.com/MystenLabs/sui/tree/main/explorer/client#readme) for browsing the TestNet transaction history: http://explorer.devnet.sui.io
+* A [Discord channel](https://discordapp.com/channels/916379725201563759/971488439931392130) for requesting test coins that can be used to pay for gas on the test network. These coins have no financial value and will disappear each time we reset the network.
+
+Many improvements to the Sui DevNet are underway, such as the ability to run full nodes and use a browser-based wallet. See the [Sui DevNet blog post](https://medium.com/mysten-labs) announcement for full details on upcoming features.
+
+## Purpose
+
+DevNet deployment features the following components of the Sui blockchain:
+
+* The [Sui Wallet command line interface (CLI)](../build/wallet.md)
+    * create and manage your private keys
+    * submit transactions for creating example NFTs
+    * call and publish Move modules
+- The Gas Faucet service
+    - transfer SUI tokens to you so you may pay for the transactions
+- [Sui Explorer](https://github.com/MystenLabs/sui/tree/main/explorer/client#readme)
+    - View transactions and objects
+
+## Prerequisites
+
+### Set up environment
+
+To use the Sui DevNet, you will need:
+
+1. Sui test coins (tokens) requested through [Discord](https://discordapp.com/channels/916379725201563759/971488439931392130)
+1. A command line terminal, as virtually everything done here is done by CLI
+1. the [`git` command line interface](https://git-scm.com/download/)
+1. The [Rust and Cargo](https://doc.rust-lang.org/cargo/getting-started/installation.html) toolchain, updated with `rustup update stable`
+1. [Sui binaries](../build/install.md#binaries) in your PATH environment variable, particularly `wallet`
+
+Remember, you can confirm the existence of a command in your PATH by running `which` followed by the command, for example:
+
+```shell
+$ which wallet
+```
+You should see the path to the command. Otherwise, reinstall.
+
+> **Tip:** To reliably test DevNet with the latest Sui binaries, re-install them at least weekly.
+
+In addition, to conduct advanced work such as publishing a Move module or making a Move call, also obtain:
+
+1. a [GitHub account](https://github.com/signup) if you don't have one already
+1. the [Sui source code](../build/install.md#source-code) installed in: `~/sui`
```

Comment:
```suggestion
1. the [Sui source code](../build/install.md#source-code) downloaded
```

I don't think installing in `~/sui` is a requirement, they just need to have the sui repo locally    @awelc to elaborate

Reply:
@sblackshear suggested this in the draft. I think we should do so in the central install docs so we might standardize across all docs. Or set a SUI_ROOT environment variable or some such.

Reply:
https://www.notion.so/Internal-Only-Sui-DevNet-Dogfooding-Guide-41d8c144897941f188d446f41411b919?d=926d0a283cae4272bf3bc91d47970960#dda2a8ffc0734f3793d13a356616564e

Reply:
Also, we can change this to a suggestion. WDYT?

Reply:
I made this a recommendation, along with the alternative environment variable

Reply:
Sounds good